### PR TITLE
Trials Project - Code Review by John

### PR DESCRIPTION
**Pull request by Rysth**
### Learning Objectives

- Learn how to do a Code Review.
- Check if semantic tags are correctly used.
- Check if HTML & CSS best practices are correctly used.
- Add a minimum of 2 comments for the pull request.

### CODE REVIEW

**Review by Rysth — COMMENTED** [review]
### Good day 👋

Nice to meet you, I'm John Palacios a Microverse Student, and today I'm going to make you a code review to enhance your project and your performance as an Excellent Student 💯.

### Changes Required 🚀

Here are a few things that going to increase your success in this project.

####  HTML

Within the **index.html** I suggest you make the next things:

- [ ] Add the **meta charset="UTF-8"** tag to specify an encoding format.
- [ ] Add the **meta name="viewport" content="width=device-width, initial-scale=1.0"** tag to specify the initial width of the web page.

Also, don't forget to use semantic tags to enhance your SEO.

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Rysth 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used GitHub inline comments and summary template 👍
Comments are descriptive with suggestion in a polite manner 👍

## Required Changes ♻️
**Kindly ensure to add at least one more inline comment in accordance to the project requirement. 👇 **

Untitled

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (Rysth) [inline index.html:1]: Please, use the correct _h1-h6_ order to enhance the SEO. Here are a few things that can help you:

-  Change the **h3 id="headline"** for a **h2** to follow the correct Heading order.

**Review by Rysth — COMMENTED** [review]
Please, use the correct _h1-h6_ order to enhance the SEO. Here are a few things that can help you:

- [ ]   Change the **h3 id="headline"** for a **h2** to follow the correct Heading order.

Also, remember to check the Linters which let you know of any problems in your files.

File: index.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
 	<head>
+		<title>John Palacios - Trials Project</title>
 		<link rel="stylesheet" href="css/style.css" />
 	</head>
 	<body>
```
Comment (Rysth) [inline index.html:7]: Please, use the correct _h1-h6_ order to enhance the SEO. Here are a few things that can help you:

- [ ]   Change the **h3 id="headline"** for a **h2** to follow the correct Heading order.

Also, remember to check the Linters which let you know of any problems in your files.

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @Rysth, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have all linters checks passed. please, make sure all linter checks are passed before asking for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AdnanOlarmmi** [comment]
Hi @Rysth,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Commented more than once.
- Kindly note that linters passing isn't a requirement for this project.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**